includes/class-wp-object-cache.php

```diff
@@ -0,0 +1,1369 @@
+<?php
+
+namespace Rhubarb\RedisCache;
```

Comment:
Let's not rename `object-cache.php` to `class-wp-object-cache.php`, because it's just a dropin, not a class part of the plugin.

Reply:
The drop-in itself is contained in a file but as `WP_Object_Cache` is a class it needs its own class file. We could copy the class file to the `wp-content` folder and include it without referencing the plugin folder.

Reply:
I rather keep it as is. We can add some PHPCS comments/rules to prevent it from generating a warning 🙈

Reply:
Actually it does not seem to generate a warning at all...
Resolved in https://github.com/rhubarbgroup/redis-cache/pull/186/commits/300d9dc6d9fd370e15c7ac6e688d2983cc879883